docs/fabric/MAS_router/mas_fifo_arb.md

```diff
@@ -17,12 +17,11 @@ A FIFO (First-In-First-Out) arbiter is a component that arbitrates between multi
 ## Main components:
 
 ### fifo
-
+- parametrize FIFO (first in-first out) 
 #### Motivation:
-
-
+- storage requests without losing anyone.  
```

Comment:
What is the motivation for using this "IP" in the fifo_arb. what its purpose in this block. why FIFO etc.